sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -271,43 +271,43 @@ export class ServiceBusClient {
   // }
 
   /**
-   * Creates a receiver for an Azure Service Bus queue's dead letter queue.
+   * Creates a receiver for an Azure Service Bus queue's dead letter queue in peekLock mode.
    *
    * @param queueName The name of the queue to receive from.
-   * @param receiveMode The receive mode to use (defaults to PeekLock)
+   * @param receiveMode The receive mode.
```

Comment:
Just curious, why don't the dead letter receivers mention the default receiveMode like the other receivers do?

Reply:
Whoops, missed those. Updated to be like the others now.